src/main/java/seedu/edulog/logic/commands/RevenueCommand.java

```diff
@@ -1,46 +1,23 @@
 package seedu.edulog.logic.commands;
 
-import seedu.edulog.commons.util.ToStringBuilder;
 import seedu.edulog.logic.commands.exceptions.CommandException;
 import seedu.edulog.model.Model;
 
 /**
  * Calculate total revenue earned
  */
-public class RevenueCommand extends Command {
+public abstract class RevenueCommand extends Command {
     public static final String COMMAND_WORD = "revenue";
     public static final String COMMAND_USAGE = COMMAND_WORD + " [paid/unpaid]"
             + ": calculate total amount of money earned/not earned"
             + "from students who have paid/not paid. Example: revenue paid";
-    public static final String COMMAND_SUCCESS = "Total revenue %s is $%d";
+    public static final String MESSAGE_SUCCESS = "Total revenue %s is $%d";
```

Comment:
I like how you followed coding standards by making static final variable names all capital letters